<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TimeRankingDialog</class>
 <widget class="QDialog" name="TimeRankingDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>550</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>时间排行榜</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: #f0f0f0;</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="labelTitle">
     <property name="font">
      <font>
       <family>Arial Unicode MS</family>
       <pointsize>24</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: #2196F3;</string>
     </property>
     <property name="text">
      <string>时间排行榜</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayoutControls">
     <item>
      <widget class="QLabel" name="labelGridSize">
       <property name="font">
        <font>
         <family>Arial</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>规格：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboBoxGridSize">
       <property name="font">
        <font>
         <family>Arial</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QComboBox {
    padding: 5px;
    border: 1px solid #ddd;
    border-radius: 4px;
    background-color: white;
}
QComboBox::drop-down {
    border: none;
}
QComboBox::down-arrow {
    image: url(:/img/down-arrow.png);
    width: 12px;
    height: 12px;
}</string>
       </property>
       <item>
        <property name="text">
         <string>3x3</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>4x4</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>5x5</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>6x6</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>7x7</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>8x8</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="labelUndo">
       <property name="font">
        <font>
         <family>Arial</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>撤销功能：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboBoxUndo">
       <property name="font">
        <font>
         <family>Arial</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QComboBox {
    padding: 5px;
    border: 1px solid #ddd;
    border-radius: 4px;
    background-color: white;
}
QComboBox::drop-down {
    border: none;
}
QComboBox::down-arrow {
    image: url(:/img/down-arrow.png);
    width: 12px;
    height: 12px;
}</string>
       </property>
       <item>
        <property name="text">
         <string>不限</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>未使用撤销</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>使用撤销</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btnRefresh">
       <property name="font">
        <font>
         <family>Arial</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
    background-color: #2196F3;
    color: white;
    border: none;
    padding: 8px;
    border-radius: 4px;
}
QPushButton:hover {
    background-color: #1976D2;
}</string>
       </property>
       <property name="text">
        <string>查询</string>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>:/img/search.png</normaloff>:/img/search.png</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTableWidget" name="tableWidget">
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QTableWidget {
    background-color: white;
    border: 1px solid #ddd;
    border-radius: 4px;
    gridline-color: #f0f0f0;
}

QTableWidget::item {
    padding: 5px;
    border-bottom: 1px solid #f0f0f0;
}

QTableWidget::item:selected {
    background-color: #e3f2fd;
}

QHeaderView::section {
    background-color: #2196F3;
    color: white;
    padding: 8px;
    border: none;
    font-weight: bold;
}

QTableWidget::item {
    border-bottom: 1px solid #f0f0f0;
}

QTableWidget::item:nth-child(odd) {
    background-color: #f9f9f9;
}</string>
     </property>
     <property name="alternatingRowColors">
      <bool>true</bool>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::SingleSelection</enum>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
     <property name="wordWrap">
      <bool>false</bool>
     </property>
     <attribute name="horizontalHeaderMinimumSectionSize">
      <number>80</number>
     </attribute>
     <attribute name="horizontalHeaderStretchLastSection">
      <bool>true</bool>
     </attribute>
     <attribute name="verticalHeaderVisible">
      <bool>false</bool>
     </attribute>
     <column>
      <property name="text">
       <string>排名</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>用户名</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>昵称</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>规格</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>用时</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>撤销</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>完成时间</string>
      </property>
     </column>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="labelStatus">
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: #666666;</string>
     </property>
     <property name="text">
      <string>选择规格和撤销功能后点击查询</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="btnBack">
     <property name="font">
      <font>
       <family>Arial Unicode MS</family>
       <pointsize>16</pointsize>
      </font>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
    background-color: #f44336;
    color: white;
    border: none;
    padding: 10px;
    border-radius: 6px;
    font-size: 16px;
}
QPushButton:hover {
    background-color: #d32f2f;
}
QPushButton:pressed {
    background-color: #b71c1c;
}</string>
     </property>
     <property name="text">
      <string>返回</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>